packages/components/src/Window.purs

```diff
@@ -0,0 +1,9 @@
+module KSF.Window (close, clearOpener) where
+
+import Prelude
+
+import Effect (Effect)
+import Web.HTML.Window (Window)
+
+foreign import close :: Window -> Effect Unit
```

Comment:
There's `close` in the latest Web.HTML, but I tried upgrading to that and got some compile errors, so I wrapped it here rather than expanding this pull too much.